_posts/developers/apps/vocabularies/2019-01-01-04_quickstart-artifacts.md

```diff
@@ -22,12 +22,12 @@ const iri = rdf.type;
 ```
 However, an issue with these libraries is that, by their nature, they are limited to just the common, well-known vocabularies. What about your specific vocabulary, designed for your app? How can you make the terms described in that vocabulary easily reusable in your code (and easily reusable in the code of others who may wish to reuse the terms you've defined in your cool vocabulary)?
 
-The LIT artifact generator (which will be made publicly available soon) is a tool that takes any RDF vocabulary as input and automatically generates a nice source-code bundle (i.e. an artifact such as a Java JAR, or a JavaScript NPM module) that you can then easily use in your application. Let's see how it works.
+The LIT artifact generator (publicly available here: https://github.com/inrupt/artifact-generator) is a tool that takes any RDF vocabulary as input and automatically generates a nice source-code bundle (i.e. an artifact such as a Java JAR, or a JavaScript NPM module) that you can then easily use in your application. Let's see how it works.
 
 ### Generate the artifact
 
 To generate a JavaScript artifact with the default options, you can use the following command:
-- `lit-artifact-generator --inputResources https://solidproject.org/assets/misc/tutorials/quickstart-obelisk.ttl --outputDirectory /path/to/a/directory --noprompt`.
+- `lit-artifact-generator generate --inputResources https://solidproject.org/assets/misc/tutorials/quickstart-obelisk.ttl --outputDirectory /path/to/a/directory --noprompt`.
```

Comment:
I'm wondering whether to remove --outputDirectory as it only  specifies the top level directory under which 
`Generated/SourceCodeArtifacts/JavaScript/GeneratedVocab/OBELISK.js` is placed.  If unspecified, the `Generated/SourceCodeArtifacts/JavaScript/GeneratedVocab/OBELISK.js` is generated in the current directory where this is run.  Would love to get your opinion.


```suggestion
`npx @inrupt/artifact-generator generate https://solidproject.org/assets/misc/tutorials/quickstart-obelisk.ttl --noprompt`.
```